### PR TITLE
Nikolai.berestevich/fix/homepage paddings

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:

-changes to decrease too big padding between " Testimonial" and "sign up "Banner in the bottom of the page
-fix "sing up" banner on the screen's width < 340px

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [x ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_signup-public.js [hunk 1]
```diff
@@ -22,6 +22,18 @@ const query = graphql`
         }
     }
 `
+const StyledSectionContainer = styled(Box).attrs({ as: 'section' })`
+    width: 100%;
+    padding:80px 0;
+    position: 'static';
+    
+    /* prettier-ignore */
+    background-color: 'white';
```
Comment (sean-binary) [inline src/components/custom/_signup-public.js:31]:
```suggestion
    padding: 80px 0;
    position: static;
    background-color: var(--color-white);
```

**Comment by njazuli-deriv** [comment]
Just my two cents, I think we should use adaptive height for this carousel, as some of the slides still leaves a gap.. for example 

image
* this one is fine

image
*notice the gap between the client's image and the bottom containers for carousel..

Maybe this [codesandbox](https://codesandbox.io/s/embla-carousel-vanilla-github-issue-113-xnne3) can help for adaptive height reference. 🙏🙏🙏

**Comment by nikolaiberestevich-deriv** [comment]
> Just my two cents, I think we should use adaptive height for this carousel

I can do it, but in this case "sing-up" component will jump up or down with every change of carusel's person . carusel's person changes every 6 seconds

**Comment by njazuli-deriv** [comment]
> > Just my two cents, I think we should use adaptive height for this carousel
> 
> I can do it, but in this case "sing-up" component will jump up or down with every change of carusel's person . carusel's person changes every 6 seconds

Okay..let's proceed 👌

**Review by njazuli-deriv — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**